app/views/coronavirus_form/offer_space_type.erb

```diff
@@ -0,0 +1,47 @@
+<% content_for :title do %><%= t('coronavirus_form.offer_space.title') %><% end %>
+<% content_for :meta_tags do %>
+  <meta name="description" content="<%= t('coronavirus_form.what_kind_of_space.title') %>" />
+<% end %>
+
+<% content_for :back_link do %>
+  <%= render "govuk_publishing_components/components/back_link", { href: previous_path } %>
+<% end %>
+
+<%= form_tag({},
+  "data-module": "track-coronavirus-form-business-offer-space",
```

Comment:
Should this be different from the `offer_space` view?

Reply:
I think so, this gets rendered as a `data-attribute` in the HTML,
So we're using html attribute syntax wich is all kebabs i believe

Reply:
Yes, yes it should. Thank you!